app/wizard.py

```diff
@@ -54,7 +57,10 @@ def __init__(self, mypkcs, myacme, oidc_provider_base_url: urllib.parse.ParseRes
 
 
 if __name__ == "__main__":
-    load_dotenv()
+    env_file_path = PROJECT_ROOT / ".env"
+    load_dotenv(dotenv_path=env_file_path)
```

Comment:
This should not be needed when inside the root folder and executing the program with `python -m app.wizard`.